test/aws/files/CentOS-OpenShift-Origin.repo

```diff
@@ -8,7 +8,7 @@ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-PaaS
 [centos-openshift-origin-testing]
 name=CentOS OpenShift Origin Testing
 baseurl=http://buildlogs.centos.org/centos/7/paas/x86_64/openshift-origin/
-enabled={{ 1 if openshift_repos_enable_testing else 0 }}
+enabled=0
```

Comment:
I've not followed the history of why `openshift_repos_enable_testing` was even here.  Do we have context?

Reply:
It was to enable the test centos paas sig repo.

Reply:
I guess we no longer need that so I think this particular change is fine.